src/renderers/materialized/materialized.text.control.tsx

```diff
@@ -21,6 +21,8 @@ export class MaterializedTextControl extends Control<ControlProps, ControlState>
                hidden={!visible}
                disabled={!enabled}
                autoFocus={uischema.options && uischema.options.focus}
+               maxlength={maxLength}
+               style={'width :' + maxLength * 15 + 'px;'}
```

Comment:
Use a calculation for the font size like shown here: [JavaScript Font Size](https://stackoverflow.com/questions/15195209/how-to-get-font-size-in-html)

Reply:
What will happen if you add something like `body { font-size: 10.0vh; }` in the `example.css`.
Will the constant width of 15px still work?

Reply:
I get it now, thanks 😄